readthedocs/core/static-src/core/js/doc-embed/rtd-data.js

```diff
@@ -0,0 +1,16 @@
+/*
+ * This exposes data injected during the RTD build into the template. It's
+ * provided via the global READTHEDOCS_DATA variable and is exposed here as a
+ * module for cleaner usage.
+ */
+
+
+var data = READTHEDOCS_DATA;
```

Comment:
Ok, I changed the logic for this now. So that the variable will not be read before `$(document).ready()` fires. I wasn't sure how to properly test this. Do you have a test-theme for this in mind where it did break?